java/arcs/core/storage/StorageProxy.kt

```diff
@@ -153,6 +161,7 @@ class StorageProxy<Data : CrdtData, Op : CrdtOperationAtTime, T>(
      * being thrown.
      */
     fun close() {
+        _crdt = null
```

Comment:
This should probably take place on the scheduler, as all other handling of the crdt model happens on the scheduler.